cmake/DaemonFlags.cmake

```diff
@@ -148,13 +148,20 @@ else()
     set(WARNMODE "no-")
 endif()
 
+# Compiler options
+option(USE_FLOAT_EXCEPTIONS "Use floating point exceptions" OFF)
 option(USE_FAST_MATH "Use fast math" ON)
 
-# Compiler options
+if (USE_FLOAT_EXCEPTIONS)
+	add_definitions(-DDAEMON_USE_FLOAT_EXCEPTIONS)
```

Comment:
That was a mistake in my environment. This is used and it works.

Reply:
I was thinking it is not needed since you could compile in the cvars regardless, but according to your other comments it is actually needed to avoid breaking the build on old Mac toolchains.